### DISCUSSION TITLE
Get array of urls

### DISCUSSION THREAD

**Discussion by kossa** [spatie/laravel-medialibrary#2115] [Q&A] [answered]
Hello,

I'm making API, I want to send with my model array of links(media), I created an accessor works fine just want to know if there is a helper for that ?

```
public function getMediaUrlsAttribute()
{
        return $this->getMedia()->map(fn ($media) => $media->getFullUrl());
}

```

**Comment by freekmurze** [comment, marked as answer]
The medialibrary doesn't do this out of the box, you should handle it in your own app.

**Comment by kossa** [comment]
I see thank you
